packages/flutter/lib/fix_data.yaml

```diff
@@ -0,0 +1,134 @@
+# Spec file for "dart fix", see <insert URL>.
```

Comment:
@bwilkerson Where can we link to here?

Reply:
We don't currently have a published version of the docs. They'll be the contents of the "User documentation" doc you've seen. I'd like to flesh them out a bit more before publishing them, but if we need them to be published sooner than that let me know.